src/goto-symex/symex_target_equation.cpp

```diff
@@ -27,7 +28,15 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "equation_conversion_exceptions.h"
 #include "goto_symex_state.h"
 
-/// read from a shared variable
+/// Transform reading from a shared variable into an SSA step. Reading
+/// \c ssa_rhs (which will thus become a left-hand side as well): we
+/// effectively assign the value stored in \c ssa_rhs by another thread to
+/// \c ssa_rhs in the memory scope of this thread.
+/// \param guard: condition to take this read event
+/// \param ssa_rhs: variable to be read from
+/// \param atomic_section_id: ID of the atomic section in which this read
+/// takes place
```

Comment:
:pick: Indent by 2 spaces for better readability. :banana:

Reply:
Done, and hopefully all the bananas.

Reply:
Thanks, I think only one space is added now but that's probably enough :slightly_smiling_face: